Student/Student/StudentAppDelegate.swift

```diff
@@ -55,10 +55,11 @@ class StudentAppDelegate: UIResponder, UIApplicationDelegate, AppEnvironmentDele
         try? AVAudioSession.sharedInstance().setCategory(.playback, mode: .default)
 
         if launchOptions?[.sourceApplication] as? String == Bundle.teacherBundleID,
-            let url = launchOptions?[.url] as? URL,
-            let components = URLComponents(url: url, resolvingAgainstBaseURL: true),
-            components.path == "student_view",
-            let fakeStudent = LoginSession.mostRecent(in: .shared, forKey: .fakeStudents) {
+           let url = launchOptions?[.url] as? URL,
+           let components = URLComponents(url: url, resolvingAgainstBaseURL: true),
+           components.path.contains("student_view"),
+           let fakeStudent = LoginSession.mostRecent(in: .shared, forKey: .fakeStudents) {
+            self.environment.k5.sessionDefaults?.isK5StudentView = components.path.contains("k5")
```

Comment:
`self.environment.k5.sessionDefaults` is equivalent to `self.environment.userDefaults`. Let's use this later one as the earlier one isn't supposed to be exposed for writing. We can change all such instances in this file.